--- usr/share/debootstrap/functions	2011-03-04 06:53:53.000000000 -0500
+++ usr/share/debootstrap/functions	2011-04-05 16:35:58.000000000 -0400
@@ -309,7 +309,7 @@
 		if [ -z "$md5" ]; then
 			[ "$dest2" = "$dest" ] || mv "$dest2" "$dest"
 			case "$typ" in
-			    gz)  gunzip "$dest" ;;
+			    gz)  /system/xbin/gunzip "$dest" ;;
 			    bz2) bunzip2 "$dest" ;;
 			esac
 			return 0
@@ -512,7 +512,7 @@
 			md="$normmd"
 		elif [ -x /bin/bunzip2 ] && [ "$bz2md" != "" ]; then
 			md="$bz2md"
-		elif [ -x /bin/gunzip ] && [ "$gzmd" != "" ]; then
+		elif [ -x /system/xbin/gunzip ] && [ "$gzmd" != "" ]; then
 			md="$gzmd"
 		fi
 		if [ "$md" != "" ]; then
@@ -542,7 +542,7 @@
 			ext="$ext $bz2md bz2"
 			md="${md:-$bz2md}"
 		fi
-		if [ -x /bin/gunzip ] && [ "$gzmd" != "" ]; then
+		if [ -x /system/xbin/gunzip ] && [ "$gzmd" != "" ]; then
 			ext="$ext $gzmd gz"
 			md="${md:-$gzmd}"
 		fi
@@ -676,9 +676,9 @@
 	    for c in $COMPONENTS; do
 		local path="dists/$SUITE/$c/binary-$ARCH/Packages"
 		local pkgdest="$TARGET/$($DLDEST pkg "$SUITE" "$c" "$ARCH" "$m" "$path")"
-		if [ -x /bin/gunzip ] && get "$m/${path}.gz" "${pkgdest}.gz"; then
+		if [ -x /system/xbin/gunzip ] && get "$m/${path}.gz" "${pkgdest}.gz"; then
 			rm -f "$pkgdest"
-			gunzip "$pkgdest.gz"
+			/system/xbin/gunzip "$pkgdest.gz"
 		elif get "$m/$path" "$pkgdest"; then
 			true
 		fi
