--- debootstrap-1.0.26+squeeze1/usr/sbin/debootstrap	2011-03-04 06:53:53.000000000 -0500
+++ debootstrap-1.0.26+squeeze1/usr/sbin/debootstrap	2011-04-05 15:13:01.000000000 -0400
@@ -419,9 +419,10 @@
 		error 1 NEEDROOT "debootstrap can only run as root"
 	fi
 	# Ensure that we can create working devices and executables on the target.
-	if ! check_sane_mount "$TARGET"; then
-		error 1 NOEXEC "Cannot install into target '$TARGET' mounted with noexec or nodev"
-	fi
+# this doesn't seem to work properly on Android, so comment it out
+#	if ! check_sane_mount "$TARGET"; then
+#		error 1 NOEXEC "Cannot install into target '$TARGET' mounted with noexec or nodev"
+#	fi
 fi
 
 if [ ! -e "$SCRIPT" ]; then
